tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -1282,7 +1282,8 @@
         parameters: {
           script: script,
           timeout: timeout,
-          args: args
+          args: args,
+          sandbox: 'simpletest',
```

Comment:
Needs a comment here about why we need the `simpletest` sandbox for `executeJSScript`